app/attendees/[id]/page.tsx

```
@@ -115,6 +119,21 @@ export default function AttendeePage() {
             </div>
           </section>
 
+          <section
+            id="attendee-confirmation-status"
+            className="flex items-center gap-2"
+          >
+            {attendee.assisted ? (
+              <>
+                <Check className="text-green-500" /> <>Asistente confirmado</>
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)